tutorials/performance/using_servers.rst

```diff
@@ -119,6 +119,32 @@ This is an example of how to create a sprite from code and move it using the low
         var xform = Transform2D().rotated(deg_to_rad(45)).translated(Vector2(20, 30))
         RenderingServer.canvas_item_set_transform(ci_rid, xform)
 
+ .. code-tab:: csharp
+
+    public partial class MyNode2D : Node2D
+    {
+        // RenderingServer expects references to be kept around. GC will otherwise remove your objects sooner or later...
+        Texture2D texture;
+        // We keep the Rid for later access, for example if we want to remove (clear) the item later.
+        Rid ci_rid;
```

Comment:
The GDScript example doesn't keep a reference to this RID.

```suggestion
```

Reply:
it's important for the next snipped to work, that's why i kept it